app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.java

```diff
@@ -58,16 +59,21 @@
     private boolean ignoreActionFocus = false;
     private String webViewTitle = null;
     private String webViewURL = null;
+    private PasswordEntry lastPassword;
+    private long lastPasswordDate;
 
-    public final class Constants {
-        public static final String TAG = "Keychain";
+    final class Constants {
+        static final String TAG = "Keychain";
+        static final long MAX_PASSWORD_ENTRY_LIFETIME = 45 * 1000;
```

Comment:
It's `general_show_time`, we should rename it as it's used for visibility and clipboard timeouts.